### PR TITLE
Add Pimax Platform SDK

### PR DESCRIPTION
**Pull request by mikeskydev**
Adds a new Pimax build target and includes/enables Pimax SDK on a custom SteamVR build.

### CODE REVIEW

**Review by mikeage — APPROVED** [review]
LGTM

**Comment by merulaxr** [comment]
I added the CI in the end!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -523,6 +553,35 @@ jobs:
...
+    needs: [configuration, build]
+    runs-on: windows-latest
+    env:
+      PIMAX_APP_ID: ${{ secrets.PIMAX_APP_ID }}
+      PIMAX_USERNAME: ${{ secrets.PIMAX_USERNAME }}
+      PIMAX_PASSWORD: ${{ secrets.PIMAX_PASSWORD }}
+    if: |
+      github.event_name == 'push' &&
+      github.repository == 'icosa-gallery/open-brush' &&
+      (contains(github.ref, 'refs/tags/v'))
```
Comment (mikeage) [inline .github/workflows/build.yml:567]: Does Pimax have a concept of release channels?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -124,6 +124,9 @@ jobs:
           - name: Windows SteamVR
             targetPlatform: StandaloneWindows64
             vrsdk: SteamVR
+          - name: Windows Pimax
```
Comment (mikeage) [inline .github/workflows/build.yml:127]: BTW, note that there will be a slight conflict when merging this into the xr_v2 branch, but it should be easy enough to fix

Comment (merulaxr) [reply .github/workflows/build.yml:127]: Yup, would have preferred to target that branch but gotta be stable! Will fix in due course.

File: .github/workflows/build.yml (hunk 1 above)
Comment (merulaxr) [reply .github/workflows/build.yml:567]: Not at this time, the tool only serves to upload the builds

Comment (mikeage) [reply .github/workflows/build.yml:567]: oof. Ok.

**Pull request merged**